src/interface/desktop/assets/khoj.css

```diff
@@ -89,7 +89,36 @@ nav.khoj-nav {
     grid-auto-flow: column;
     grid-gap: 32px;
     justify-self: right;
-}
+    align-items: center;
+  }
+  
+  .khoj-status-box {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 50px;
+    gap: 5px;
+  }
+  
+  .khoj-status-box .khoj-status-connected {
+    height: 13px;
+    width: 13px;
+    border-radius: 50%;
+    background-color: rgb(90, 235, 90);
+  }
+  .khoj-status-box .khoj-status-not-connected {
+    height: 13px;
+    width: 13px;
+    border-radius: 50%;
+    background-color: rgb(235, 90, 90);
+  }
+  
+  .khoj-status-box .khoj-status-text {
+    display: none;
+  }
+  .khoj-status-box:hover .khoj-status-text {
+    display: block;
+  }
```

Comment:
We _generally_ try to use multiples of 4 for any of the pixel values, like padding, gap, width. It helps give the overall layout some more subtle unity. Could you update these to fit that? For example, 13px -> 12px, or 5px -> 8px. Feel free to round up or down depending on the use.